roles/reset/tasks/main.yml

```diff
@@ -52,7 +52,7 @@
     daemon_reload: yes
 
 - name: Remove linux-modules-extra-raspi
-  apt: name=linux-modules-extra-raspi state=absent
+  package: name=linux-modules-extra-raspi state=absent
```

Comment:
I wonder whether our default scenario should use multiple distributions. If Debian, Ubuntu and CentOS are supported, we could at least convert one node over to CentOS?

Reply:
i am not sure how to answer your question. In my case i was configuring a single node cluster on CentOS7 and everything went smoothly. I just had to change this module and then adjust the transient service unit test with the rescue block, which i had forgot to push: https://github.com/techno-tim/k3s-ansible/pull/79/commits/48c8372317440e6a666eb2cf4c60c149bd4961d7

Reply:
We do automated testing of the playbook in CI, using [molecule](https://molecule.readthedocs.io). The tests should cover the offered functionality as best as possible. Currently, we only test on Ubuntu VMs, so we will not notice in CI if we break CentOS. So I'm wondering whether it makes sense to use a hybrid cluster (as in Ubuntu _and_ CentOS VMs) for the default scenario. @timothystewart6, thoughts?

---

But since this PR also introduces a new feature (immediate change of IP pools), it should be tested too IMO.
Technically speaking, we could choose a scenario and add an extra [side_effect](https://molecule.readthedocs.io/en/latest/configuration.html#molecule.scenario.Scenario) (to provision with different IP addresses) and and verify step after the first verify step.

Reply:
This code was forked from another code base but I am puzzled as to why this would not work.  -p is a way to pass an option and remove this along with the options means that the service will not sleep nor restart if it fails, something you probably want.  I am puzzled as to why this doesn't work for CentOS, especially for something as core as systemd.  Would be nice at some point to test this on all supported OS, but that aiside @dberardo-com  which version of CentOS are you using?  What does it print when you run `man systemd` ?  I assume it supports `-p`.  

The workaround of trying again without parameters is better than not working at all, but I would like to figure out why it's not working.

Reply:
> I wonder whether our default scenario should use multiple distributions. If Debian, Ubuntu and CentOS are supported, we could at least convert one node over to CentOS?

I hinted at that in my last comment but it would be cool to have one of each distribution :)